xbmc/windowing/X11/WinSystemX11.cpp

```diff
@@ -521,6 +523,11 @@ void CWinSystemX11::NotifyAppFocusChange(bool bGaining)
   }
   if (!bGaining)
     m_bIgnoreNextFocusMessage = false;
+  else
+  {
+    Atom utf8_string = XInternAtom(m_dpy, "UTF8_STRING", False);
+    XConvertSelection(m_dpy, XA_PRIMARY, utf8_string, None, m_glWindow, CurrentTime);
```

Comment:
I don't think we should blindly try to convert anything on AppFocucChange. Further I think we want to copy from clipboard, not primary. We may also copy from Kodi to Kodi.